devito/mpi/routines.py

```diff
@@ -555,10 +555,10 @@ def _make_sendrecv(self, f, hse, key, msg=None):
         count = reduce(mul, sizes, 1)
         rrecv = Byref(FieldFromPointer(msg._C_field_rrecv, msg))
         rsend = Byref(FieldFromPointer(msg._C_field_rsend, msg))
-        recv = Call('MPI_Irecv', [bufs, count, Macro(dtype_to_mpitype(f.dtype)),
-                                  fromrank, Integer(13), comm, rrecv])
-        send = Call('MPI_Isend', [bufg, count, Macro(dtype_to_mpitype(f.dtype)),
-                                  torank, Integer(13), comm, rsend])
+        recv = IrecvCall('MPI_Irecv', [bufs, count, Macro(dtype_to_mpitype(f.dtype)),
```

Comment:
instead of harcoding `'MPI_Irecv'` here, I'd just change the corresponding class as

```
class IrecvCall(Call):

    def __init__(self, parameters):
          super(IrecvCall, sell).__init__('MPI_Irecv', parameters)
```

Reply:
I'll push that together with the next changes.